apps/api/src/database/migrations/20230802104058_add_status_reference_columns_s51_advice/migration.sql

```diff
@@ -0,0 +1,21 @@
+BEGIN TRY
+
+BEGIN TRAN;
+
+-- AlterTable
+ALTER TABLE [dbo].[S51Advice] ADD [publishedStatus] NVARCHAR(1000) NOT NULL CONSTRAINT [S51Advice_publishedStatus_df] DEFAULT 'not_checked',
+[redactedStatus] NVARCHAR(1000) NOT NULL CONSTRAINT [S51Advice_redactedStatus_df] DEFAULT 'not_redacted',
```

Comment:
in DocumentVersion, redactedStatus allows null, and no default value - so only changes if the user explicitly sets it to redacted, or not_redacted.  Should we be consistent and do the same thing here?